Mathlib/Algebra/Order/Pointwise.lean

```diff
@@ -106,15 +121,30 @@ section One
 
 variable [One α]
 
-@[simp, to_additive]
-theorem Sup_one : supₛ (1 : Set α) = 1 :=
+-- Porting note: Use something like `@[to_additive (attr := simp)]` instead.
+-- However, this makes a simpNF error like:
+--  `supₛ_one` can be proved like `by simp only [@supₛ_zero]`
+-- So the `simp` attribute is written below the definition.
+@[to_additive]
+theorem supₛ_one : supₛ (1 : Set α) = 1 :=
```

Comment:
I think we should try to solve this in a different way. I asked on zulip: https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/simpNF/near/321179015

Reply:
I can't reproduce this `simpNF` error.

Reply:
@eric-wieser The attributes below should be removed to reproduce the error, with the following code:

Reply:
It seems like swapping the place of two sections `CompleteLattice` and `ConditionallyCompleteLattice` fixes the issue. Thanks for reviewing, I could fix it.